readme.md

```diff
@@ -273,7 +273,7 @@
 - [Coq](https://github.com/coq-community/awesome-coq#readme) - Formal language and environment for programming and specification which facilitates interactive development of machine-checked proofs.
 - [V](https://github.com/vlang/awesome-v#readme) - Simple, fast, safe, compiled language for developing maintainable software.
 - [Zig](https://github.com/catdevnull/awesome-zig#readme) - General-purpose programming language and toolchain for maintaining robust, optimal, and reusable software.
-- [Move](https://github.com/MystenLabs/awesome-move#readme) - Domain-specific programming language for writing safe smart contracts.
+- [Move](https://github.com/sui-foundation/awesome-sui#move-ides) on Sui - Domain-specific programming language for writing safe smart contracts works on Sui.
```

Comment:
Not sure if referencing same repositories is allowed, but I guess it is open for discussion